lib/napa/grape_extenders.rb

```diff
@@ -7,12 +7,12 @@ def self.extended(modified_class)
       if defined?(::ActiveRecord)
         modified_class.rescue_from ::ActiveRecord::RecordNotFound do |e|
           err = Napa::JsonError.new(:record_not_found, 'record not found')
-          Napa::Logger.logger.debug Napa::Logger.response(404, {}, err)
+          Napa::Logger.logger.debug Napa::Logger.response_format(404, {}, err)
```

Comment:
Do we need to check the ```Napa::Logger.config.format``` here and call ```basic_response_format ``` if it's set to ```:basic```? Should we have a helper method to handle this since we're doing so in a couple of different places (see https://github.com/bellycard/napa/pull/213/files#diff-d080f292ba7dad8a943d0d03f9c81349R53)?